griptape/engines/query/vector_query_engine.py

```diff
@@ -23,6 +23,8 @@ class VectorQueryEngine(BaseQueryEngine):
     )
     template_generator: J2 = field(default=Factory(lambda: J2("engines/query/vector_query.j2")), kw_only=True)
```

Comment:
While I do like the granularity of being able to override specific bits of the prompt, I'm not sure this change is necessary since users can override `template_generator` to completely change the prompt.

@andrewfrench @vasinov do you have any suggestions for how we might improve this?